contracts/Contribution.sol

```diff
@@ -111,7 +111,7 @@ contract Contribution is Controlled, TokenController {
   }
 
   // ETH-AIX exchange rate
-  function exchangeRate() constant public initialized returns (uint256 rate){
+  function exchangeRate() constant public initialized returns (uint256 rate) {
```

Comment:
What's recommended for the returns value, I noticed that Andres removed the variable names from his.

`returns (uint256)` vs `returns (uint256 rate)`

Reply:
if the rate is not used, yes. I also don't use variable names

Reply:
I like naming only because then I don't have to return anything, just make sure that I assign the value to the variable.